### ISSUE TITLE
Error Vulkan Driver

### ISSUE THREAD

**Issue by ToniBC** [libretro/RetroArch#11408]
Hello, for 2 days errors have been appearing with the Vulkan driver in Retroarch.

First, almost all the cores that use Vulkan stopped working and now it does not allow Retroarch to enter. Before, everything worked without problems.

I have tried with a clean copy of Retroarch, it enters without problems, but when I select as Vulkan driver, you close and it does not start anymore. With GL it works without problems.

All this on updated x64 Windows and updated Nvidia GTX 970 graphics.

2 days ago everything worked fine, but suddenly it stopped working. I see that I am not the only one, because another user left a post in the forum with the same problem.

https://forums.libretro.com/t/retroarch-crash-to-desktop-if-i-use-vulkan-video-driver/30351/4

If you activate the LOG in debug mode, it always gives the following error, when loading some core with Vulkan before and now when trying to enter the interface alone.

![Error Vulkan](https://user-images.githubusercontent.com/45365478/94998514-86ded480-05b2-11eb-8c8b-d9b9ee7a02cd.png)

I have reinstalled Windows thinking that it would be the problem, everything clean and still does not work, when it has been 2 or 3 days everything without problems. Something was updated in Windows or in the drivers that affects RA.

Thank you.

**Comment by twinaphex** [comment]
Do a bisect, and also the link you posted is broken.

Vulkan still works fine on my desktop PC.

**Comment by ToniBC** [comment]
The link is already fixed, it is a forum message.

Yesterday I got it to work by returning Windows to September 28, which was the only save point I had and it worked again, but the next day, that is, yesterday, the same again.

I thought it would be Windows thing or something weird, but reinstalled Windows and nothing, same error.

It seems that something was updated that affects Vulkan and certain settings. By default, as Retroarch starts with GL, many do not change anything and continue using GL, they do not see the error.

The only error it gives is that, with the default configuration, that is, delete retroarch.cfg, enter, change to Vulkan, activate LOGS, exit, and when entering again, error and closes. That same error appeared 2 days ago when I loaded Flycast, Parallel, etc ... If you put GL it works without problems.

**Comment by twinaphex** [comment]
Like I asked before, can you 'bisect' it? What I mean by that is, go back commit by commit until you find the commit that broke it. 

I cannot do this since I cannot reproduce this error.

**Comment by twinaphex** [comment]
So I guess a followup question to that is, what changed in your setup? Did you upgrade RetroArch, did you upgrade your video driver? What exactly happened that caused this?

**Comment by ToniBC** [comment]
I already tried everything, I only have one doubt. Could it be a hardware problem?

That is, the card works without problems in DX11 and 12, passing tests as 3DMARK, but in Vulkan it gives an error, it does not pass the test. Now the vulkan api stopped working in everything, not only retroarch, other emulators and programs crashed.

I don't know if you understand this, but could it be that the GPU is a bit broken and only affects Vulkan? It is something that happened suddenly and the truth is strange.

**Comment by DfactorLongplays** [comment]
I've had the same issue, as noted above it's not specifically caused or related to RetroArch.

In my case what did help is uninstalling Bandicam. Funny thing is, I don't remember updating it recently, but apparently there's software conflict going on and no matter which drivers I'd reinstall vulkaninfo would fail.

**Comment by ToniBC** [comment]
It seems that it is a Windows problem, I reinstalled Windows, obviously everything updated to 2004 and with all the updates and drivers and it continues to fail. Before that, using the restorer, I went back to September 28 and it worked again, the next day it stopped working, I suppose something was updated.

It is not related to Retroarch, so I close this, because other emulators and programs all crash on Vulkan. Since I don't have another more modern card, I can't try.

**Comment by ToniBC** [comment]
It seems that it is a Windows problem, I reinstalled Windows, obviously everything updated to 2004 and with all the updates and drivers and it continues to fail. Before that, using the restorer, I went back to September 28 and it worked again, the next day it stopped working, I suppose something was updated.

It is not related to Retroarch, so I close this, because other emulators and programs all crash on Vulkan. Since I don't have another more modern card, I can't try.
